### PR TITLE
Fix problem links

### PR DESCRIPTION
**Pull request by sdake**
Some of these changes need to go into the API repo as well.

### CODE REVIEW

File: content/pt-br/news/releases/0.x/announcing-0.1/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Google, IBM and Lyft joined forces to create Istio from a desire to provide a re
...
 
-**Secure by default**: It is a common fallacy of distributed computing that the network is secure. Istio enables operators to authenticate and secure all communication between services using a mutual TLS connection, without burdening the developer or the operator with cumbersome certificate management tasks. Our security framework is aligned with the emerging [SPIFFE](https://spiffe.github.io/) specification, and is based on similar systems that have been tested extensively inside Google.
+**Secure by default**: It is a common fallacy of distributed computing that the network is secure. Istio enables operators to authenticate and secure all communication between services using a mutual TLS connection, without burdening the developer or the operator with cumbersome certificate management tasks. Our security framework is aligned with the emerging [SPIFFE](https://spiffe.o/) specification, and is based on similar systems that have been tested extensively inside Google.
```
Comment (jhonmike) [inline content/pt-br/news/releases/0.x/announcing-0.1/index.md]:
```suggestion
**Secure by default**: It is a common fallacy of distributed computing that the network is secure. Istio enables operators to authenticate and secure all communication between services using a mutual TLS connection, without burdening the developer or the operator with cumbersome certificate management tasks. Our security framework is aligned with the emerging [SPIFFE](https://spiffe.io/) specification, and is based on similar systems that have been tested extensively inside Google.
```

**Comment by sdake** [comment]
@jhonmike can you please sign the google CLA so this PR can merge.

Cheers
-steve

**Comment by jhonmike** [comment]
@googlebot I fixed it.

@sdake I had changed my default github email, sorry!

**Comment by sdake** [comment]
> @googlebot I fixed it.
> 
> @sdake I had changed my default github email, sorry!

@jhonmike welcome aboard the Istio party. Thanks for acting quickly on my request. If you need anything, feel free to reach out - github is better for dev and design discussion, but I'd recommend attempting to join slack if you haven't already.

Cheers
-steve

**Comment by sdake** [comment]
> Spiffe change is also in my PR: #6379 but I can't approve that one so will approve this one.

@ericvn my bad for not checking the queue first - was in a bit of a rush ;)

Cheers
-steve

**Pull request merged**